src/widgets.js

```diff
@@ -591,11 +722,50 @@ define([
         return $el;
     }
 
+    function replaceOuputRef(form, value, withClose, noOutput) {
+        var template = "<output value=\"" + value + "\" />";
+        if (noOutput) {
+            template = value;
+        }
+        var v = value.split('/'),
+            dispValue = v[v.length-1],
+            mug = form.getMugByPath(value),
+            icon = mug ? mug.options.icon: 'fcc fcc-fd-external-case',
```

Comment:
Has there been a `getCaseByPath` method created? Would be nice to be able to pull info from that